server/controllers/signup.js

```diff
@@ -0,0 +1,19 @@
+const addUser = require('../database/queries/addUser')
+const bcrypt = require('bcrypt')
+
+exports.signUp = (request, response) => {
+  const { email, name, password } = request.body
+
+  bcrypt.hash(password, 10, (err, hash) => {
```

Comment:
Same thing as comment below, would be great if this could be refactored to use promises.